docs/src/pages/[platform]/getting-started/create-react-app/index.page.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Create React App
+metaTitle: Create React App
+description: How to get started using Amplify UI with Create React App
+metaDescription: How to get started using Amplify UI with Create React App
+supportedFrameworks: react
+---
+
+import { Fragment } from '@/components/Fragment';
+
+<Fragment>{({ platform }) => import(`./react.mdx`)}</Fragment>
```

Comment:
Since the Create-React-App page is obviously only supported by React, it's not really necessary to have an `index.page.mdx` with a hard-coded fragment. I just defaulted to the same pattern that is used throughout the docs.